### DISCUSSION TITLE
Full health mod

### DISCUSSION THREAD

**Discussion by enoslayd** [ppy/osu#13170] [Ideas]
**Describe the new feature:** can compliment #9584. a mod for taiko (or other rulesets if applicable) that requires players to fully fill the healthbar before the beatmap ends. if the health is not fully filled, the play will fail even if the health is already in the passing mark.

**Comment by yellowsink** [comment]
I'm interested in giving this a try actually. Might have a look this evening.

**Reply by TheFearlessDeath500** [reply]
I second this.

**Comment by peppy** [comment]
Did you mean to say "before the end" rather than "until the end"?

**Comment by enoslayd** [comment]
Before sorry
